joint_limits:
  Joint2_1:
    angle: [-2.3562, 4.7124]  # -2.356194490192345, 4.71238898038469
    velocity: 10
    effort: 300

  Joint2_2:
    angle: [-2.0944, 1.5708]  # -2.0943951023931953, 1.5707963267948966
    velocity: 10
    effort: 300

  Joint2_3:
    angle: [-2.5307, 2.5307]  # -2.530727415391778, 2.530727415391778
    velocity: 10
    effort: 300

  Joint2_4:
    angle: [-3.1416, 3.1416]  # -3.141592653589793, 3.141592653589793
    velocity: 10
    effort: 100

  Joint2_5:
    angle: [-1.9199, 1.9199]  # -1.9198621771937623, 1.9198621771937623
    velocity: 10
    effort: 100

  Joint2_6:
    angle: [-3.1416, 3.1416]  # -3.141592653589793, 3.141592653589793
    velocity: 10
    effort: 100

  Joint3_1:
    angle: [-2.3562, 4.7124]  # -2.356194490192345, 4.71238898038469
    velocity: 10
    effort: 300

  Joint3_2:
    angle: [-2.0944, 1.5708]  # -2.0943951023931953, 1.5707963267948966
    velocity: 10
    effort: 300

  Joint3_3:
    angle: [-2.5307, 2.5307]  # -2.530727415391778, 2.530727415391778
    velocity: 10
    effort: 300

  Joint3_4:
    angle: [-3.1416, 3.1416]  # -3.141592653589793, 3.141592653589793
    velocity: 10
    effort: 100

  Joint3_5:
    angle: [-1.9199, 1.9199]  # -1.9198621771937623, 1.9198621771937623
    velocity: 10
    effort: 100

  Joint3_6:
    angle: [-3.1416, 3.1416]  # -3.141592653589793, 3.141592653589793
    velocity: 10
    effort: 100

floating_joint:
  position:  # 位置限制 (x, y, z)
    min: [0, -0.3, 0.24]
    max: [0.8, 0.3, 0.6]
  rotation:
    euler_min: [-45, 90, 0]  # 对应 (X, Y, Z) 轴最小角度
    euler_max: [45, 160,  0]  # 对应 (X, Y, Z) 轴最大角度

